config/routes.rb

```diff
@@ -1,9 +1,14 @@
 Rails.application.routes.draw do
+  # get 'index/new'
+
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  # root 'welcome#index'
+  root 'index#new'
+    get '/login' => 'sessions#new'
+    post '/login' => 'sessions#create'
```

Comment:
Inconsistent indentation detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.